src/components/inputs/TextInput.js

```diff
@@ -65,13 +65,33 @@ export default class TextInput extends BaseInput {
      * transform function executed on value and return transformed value
      */
     transformer: PropTypes.func,
+    /**
+     * Fixed title that will displayed above the input (note: floatingPlaceholder MUST be 'false')
+     */
+    title: PropTypes.string,
+    /**
+     * The title's color
+     */
+    titleColor: PropTypes.string,
+    /**
+     * Limits the maximum number of characters that can be entered
+     */
+    maxLength: PropTypes.number,
+    /**
+     * should the input display a character counter (only when passing 'maxLength')
+     */
+    characterCounter: PropTypes.bool,
```

Comment:
let's give it a name that identify it as boolean prop like showCharacterCounter

Reply:
ok